Source/FLHookPluginSDK/headers/plugin.h

```diff
@@ -250,12 +250,15 @@ enum PLUGIN_MESSAGE
 	DSACE_SPEED_EXCEPTION = 41,
 	CUSTOM_BASE_BEAM = 42,
 	CUSTOM_BASE_IS_DOCKED = 43,
-	CUSTOM_BASE_LAST_DOCKED = 46,
 	CLIENT_CLOAK_INFO = 44,
 	COMBAT_DAMAGE_OVERRIDE = 45,
+	CUSTOM_BASE_LAST_DOCKED = 46,
 	CUSTOM_JUMP = 47,
-	CUSTOM_IS_IT_POB = 50,
 	CUSTOM_REVERSE_TRANSACTION = 48,
+	CUSTOM_JUMP_CALLOUT = 49,
```

Comment:
Nothing seems to actually receive these messages?

Reply:
The code for that was developed as part of the HyperJump rework. It won't work without it, but I guarantee it's there.
Otherwise we'd need a lot of duplicated code across branches that'd need to be synchronized.